huaweicloud/resource_huaweicloud_lb_loadbalancer.go

```diff
@@ -208,7 +220,11 @@ func resourceLoadBalancerV2Read(d *schema.ResourceData, meta interface{}) error
 	}
 
 	// fetch tags
-	if resourceTags, err := tags.Get(elbClient, "loadbalancers", d.Id()).Extract(); err == nil {
+	elbV2Client, err := config.ElbV2Client(GetRegion(d, config))
+	if err != nil {
+		return fmt.Errorf("Error creating HuaweiCloud elb client: %s", err)
+	}
```

Comment:
How can I know the Error happened where in `ElbV2Client` or in `Elb_V2Client`.

Reply:
fixed

Reply:
fixed